### ISSUE TITLE
NameError (wrong constant name Db-migration):

### ISSUE THREAD

**Issue by nodesocket** [motor-admin/motor-admin#2]
After I run the setup wizard and successfully connect to my PostgreSQL database for the first time I am getting an error:

```bash
⚡ Motor::Admin is starting under /

Puma starting in single mode...
* Puma version: 5.5.2 (ruby 2.7.1-p83) ("Zawgyi")
*  Min threads: 20
*  Max threads: 20
*  Environment: production
*          PID: 11931
* Listening on http://0.0.0.0:3000
Use Ctrl-C to stop
I, [2021-11-02T21:53:32.126898 #11931]  INFO -- : [03f58b53-9647-49c2-95ec-36a3f38955d3] {"method":"GET","path":"/sign_in","status":302,"duration":6.27,"view":0.0,"db":1.32,"fwd":"127.0.0.1","uid":null}
I, [2021-11-02T21:53:32.142927 #11931]  INFO -- : [07649f6e-dfd7-49a7-bba9-5d97f904991b] {"method":"GET","path":"/setup","status":200,"duration":11.36,"view":4.97,"db":0.61,"fwd":"127.0.0.1","uid":null}
I, [2021-11-02T21:53:49.581512 #11931]  INFO -- : [e6ae59d4-500a-4bbd-9e74-ebdab4021f7b] {"method":"POST","path":"/api/setup","status":200,"duration":360.5,"view":0.11,"db":7.72,"fwd":"127.0.0.1","uid":1}
I, [2021-11-02T21:54:33.406803 #11931]  INFO -- : [d5759736-9fd6-411f-9d64-75ca6178e16a] {"method":"POST","path":"/api/verify_db_connection","status":200,"duration":98.2,"view":0.0,"db":2.44,"fwd":"127.0.0.1","uid":1}
I, [2021-11-02T21:54:33.432325 #11931]  INFO -- : [95012639-f5ec-4a81-8379-3bf5673df696] {"method":"POST","path":"/api/encrypted_configs","status":200,"duration":15.73,"view":0.31,"db":2.49,"fwd":"127.0.0.1","uid":1}
I, [2021-11-02T21:54:33.666596 #11931]  INFO -- : [d6af41d9-60e4-4d05-a235-0d5ece261536] {"method":"GET","path":"/","status":500,"duration":220.85,"view":0.0,"db":72.91,"fwd":"127.0.0.1","uid":1}
F, [2021-11-02T21:54:33.666994 #11931] FATAL -- : [d6af41d9-60e4-4d05-a235-0d5ece261536]
[d6af41d9-60e4-4d05-a235-0d5ece261536] NameError (wrong constant name Db-migration):
[d6af41d9-60e4-4d05-a235-0d5ece261536]
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:87:in `const_set'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:87:in `block in define_models'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:70:in `each'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:70:in `filter_map'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:70:in `define_models'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:46:in `block in call'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:43:in `synchronize'
[d6af41d9-60e4-4d05-a235-0d5ece261536] lib/motor/define_ar_models.rb:43:in `call'
[d6af41d9-60e4-4d05-a235-0d5ece261536] config/initializers/set_db_connection_and_define_ar_models.rb:33:in `set_db_connection_and_define_ar_models'
```

**Comment by omohokcoj** [comment]
@nodesocket thanks for reporting this issue - it has been fixed in the latest 0.2.18 version - hope you can give it another try 😃

**Comment by nodesocket** [comment]
@omohokcoj worked thanks.
